tests/Twig/Extension/SonataAdminExtensionTest.php

```diff
@@ -2968,10 +2963,20 @@ public function testCanonicalizedLocaleForMoment(?string $expected, string $orig
 
     public function testIsGrantedAffirmative(): void
     {
-        $this->assertTrue(
-            $this->twigExtension->isGrantedAffirmative(['foo', 'bar'])
-        );
-        $this->assertTrue($this->twigExtension->isGrantedAffirmative('foo'));
+        $this->securityChecker->method('isGranted')->willReturnCallback(static function (string $role): bool {
+            if ('foo' === $role) {
+                return false;
+            }
+
+            if ('bar' === $role) {
+                return true;
+            }
+
+            return false;
+        });
```

Comment:
It's way cleaner your way:
```suggestion
       $this->securityChecker->method('isGranted')->willReturnMap([
            ['foo', null, false],
            ['bar', null, true],
        ]);
```